File: uHunt/src/easy/One_Two_Three.java
package easy;

import java.io.BufferedReader;
import java.io.File;
import java.io.FileReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class One_Two_Three
{
	private static BufferedReader br;
	private static boolean test=true;

	public static void main(String[] args) throws IOException 
	{
		if(test)br=new BufferedReader(new FileReader(new File("./data/easy/One_Two_Three.in")));
		else br=new BufferedReader(new InputStreamReader(System.in));
		solve();
		br.close();
	}

	private static void solve() throws IOException
	{
		char[] one={'o','n','e'} ,letras;
		int casos=Integer.parseInt(br.readLine()), num1=0;
		
		while(casos--!=0)
		{
			letras=br.readLine().toCharArray();
			
			if(letras.length==5) System.out.println(3);
			else
			{
				num1=0; 
				for (int j=0; j<letras.length; j++) if(letras[j]==one[j]) num1++;
				if(num1>=2) System.out.println(1);
				else System.out.println(2);
			}
		}
	}
	
}

File: uHunt/C++src/easy/One_Two_Three.cpp
using namespace std;
#include<cstdio>
#include<cstring>

bool test=true;

int main()
{
    if(test) freopen("../../data/easy/One_Two_Three.in","r",stdin);

    char one[]={'o','n','e'};
    char linea[6];
    int casos, tam, num1;
    scanf("%d",&casos);

    while(casos--)
    {
        scanf("%s",linea);
        tam=strlen(linea);

        if(tam==5) printf("3\n");
        else
        {
            num1=0;
            for(int j=0;j<tam;j++)if(linea[j]==one[j]) num1++;
            if(num1>=2) printf("1\n");
            else printf("2\n");
        }

    }

    return 0;
}
